go.mod

```diff
@@ -4,13 +4,13 @@ go 1.17
 
 require (
 	github.com/cosmos/gorocksdb v1.2.0
-	github.com/dgraph-io/badger/v2 v2.2007.2
+	github.com/dgraph-io/badger/v2 v2.0.1-rc1.0.20210105060133-d6666aecfdc3
```

Comment:
This seems a bit sus—why are we targeting a release candidate?

It's fine to use whatever versions for benchmarking, but I think for a library shared by a lot of consumers, we ought to be conservative about what we point to.

(The RocksDB thing is a notable hole in that, but I don't think we should ream that hole out any further)

Reply:
you're totally right about that, and even the benchmarks should be on released versions